LottieSample/src/androidTest/java/com/airbnb/lottie/samples/FragmentVisibilityTests.kt

```diff
@@ -323,12 +321,17 @@ class FragmentVisibilityTests {
                         }
 
                         private fun bindLottieHolder(holder: RecyclerView.ViewHolder) {
-                            animationView = holder.itemView as LottieAnimationView
-                            (holder.itemView as LottieAnimationView).apply {
-                                repeatCount = LottieDrawable.INFINITE
-                                setAnimation(R.raw.heart)
-                                playAnimation()
-                                IdlingRegistry.getInstance().register(LottieIdlingResource(this, name = "Lottie ${Random.nextFloat()}"))
+                            if (!animationWasPlayed) {
+                                animationView = holder.itemView as LottieAnimationView
+                                (holder.itemView as LottieAnimationView).apply {
+                                    //                                    repeatCount = LottieDrawable.INFINITE
```

Comment:
In case of `LottieIdlingAnimationResource` we will wait forever if `repeatCount = LottieDrawable.INFINITE`. So I commented it out - in any case I can't see a reason to leave it here. It's not test critical condition.